README.md

```diff
@@ -37,7 +37,7 @@ function printDiskUsage(dir) {
 }
 
 run(() => printDiskUsage(process.cwd()))
-    .then(() => {}, err => { throw err; });
+    .then(() => {}, err => { console.error(err); });
```

Comment:
otherwise it create unhandlePromiseRejection in case where exception is thrown into diskUsage